messages/elixir/README.md

```diff
@@ -1,5 +1,19 @@
 # CucumberMessages
 
+This package is released at [hex.pm](https://hex.pm/packages/cucumber_messages).
+
+## Updating the generated files
+
+If you want to update the generated files, execute this command while in the elixir messages folder:
+
+```bash
+MIX_ENV=prod mix protox.generate --multiple-files --output-path=lib/cucumber_messages/generated --namespace=CucumberMessages messages.proto
+```
+
+The `--multiple-files` option is necessary if we want to support parallel compiling of modules. For more information, you can check the [protox](https://github.com/ahamez/protox) library.
+
+_Note: You do need to have `protoc` installed on your system._
+
```

Comment:
Does that still apply?
If so, it may be easier to generate the files using `make` instead of the whole described command. What do you think?

Reply:
Well it still applies, but the make process also includes the update of the messages.proto file, so best that we either 1. remove that documentation altogether or 2. give instructions as to how they can use the docker image with the make commands

I'd personally go for 1 :)

Reply:
Other implem don't have that kind of documentation, so I'd go for 1 too.

Some documentation may be required at some point, but maybe on a more wide scale in the monorepo